modules/ctconfiguration/ctconfiguration.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+/**
+ * Class CTConfiguration
+ *
+ * @property $bootstrap
+ */
+class CTConfiguration extends Module
+{
+    /**
+     * CTConfiguration constructor.
+     */
+    public function __construct()
+    {
+        $this->name    = 'ctconfiguration';
+        $this->tab     = 'front_office_features';
+        $this->version = '1.0.0';
+        $this->author  = 'Community Theme';
```

Comment:
In author list everything is in lowercase, e.g. `prestashop` `prestashop partners`